zerver/openapi/zulip.yaml

```diff
@@ -21051,6 +21076,26 @@ paths:
                           "old": 15,
                         }
                     - $ref: "#/components/schemas/GroupSettingValueUpdate"
+                can_unsubscribe_group:
+                  allOf:
+                    - description: |
+                        The set of users who have permission to unsubscribe themselves from this
+                        channel expressed as an [update to a group-setting value][update-group-setting].
+
+                        [update-group-setting]: /api/group-setting-values#updating-group-setting-values
+
+                        Note that a user must [have content access](/help/channel-permissions)
```

Comment:
I think this is incorrect as per the code now. User with metadata access and permission to administer can change this setting. And this is also inconsistent with the description below for the same setting.